lib/ts/constants.ts

```diff
@@ -15,4 +15,4 @@
 
 export const HEADER_RID = "rid";
 export const HEADER_FDI = "fdi-version";
-export const API_VERSION_FILE_PATH = "/tmp/apiversion.txt";
+export const API_VERSION_FILE_PATH = "/tmp/supertokens-apiversion.txt";
```

Comment:
- Should not be `tmp`! Should be based on `process.env` (for test) or if that is undefined, should be `/temp`
- Also, change the variable name to be somehting like `SERVERLESS_CACHE_API_VERSION_FILE_PATH`

Reply:
okay